### ISSUE TITLE
👋 Looking for maintainers 

### ISSUE THREAD

**Issue by karbassi** [karbassi/netatmo#66]
Hi everyone! 👋 

There's a lot of exciting, new things happening in my life (can't announce yet)! 🎉 

I've had a blast working on this project since it's inception, but it's time to allow others to step in and take the lead.

I'll be around to help merge, give feedback, etc., however, I don't have the time to fully maintain the project.

**I'm looking for new people to maintain the library and take over the project.**

I'll provide guidance and onboard you (if you want).

What the project needs is someone with JS (maybe TS 👍 ) knowledge who also has access to one or more Netatmo devices. If that's you and you're interested, let me know.

In the meantime, be aware that I won't work on the open issues myself.

**Comment by friedrith** [comment]
Hello ! I am interested in this project! I would like to participate in particular to add new functions related to security features in netatmo. I will propose first PR soon.

**Comment by karbassi** [comment]
@floetenbaer I look forward to your PR and hopefully maintaining the project.

**Comment by friedrith** [comment]
In fact your project is implemented with module `request` which is now deprecated. Do you authorize me to propose several PR to switch it to another HTTP client library and add unit tests? I will do it step by step and without breaking changes. I know the challenges to maintain public library since I am maintaining my own library https://www.npmjs.com/package/node-wifi

**Comment by karbassi** [comment]
@friedrith Yes, please propose several PRs with each one being one "thing". I'll be here to review, provide feedback, and merge the changes with you along the way.

Maybe create a `3.0.0` roadmap issue with a list of changes you believe need to be done. That way we can track the progress towards `3.0.0`. Thoughts?

**Comment by friedrith** [comment]
I didn't see your 3.0.0 roadmap but I think that almost everything is already there.

**Comment by friedrith** [comment]
The only things that are missing are related to CI and code style (prettier, eslint, unit tests and github actions) IHMO

**Comment by friedrith** [comment]
Are you working on switching to axios or I can propose a first small PR?

**Comment by karbassi** [comment]
> I didn't see your 3.0.0 roadmap but I think that almost everything is already there.

I didn't create one. I was suggesting you create a `3.0.0 roadmap` issue. A post that will detail out your plans for the next version. 

For example, _replacing `request` with something else_. 

> The only things that are missing are related to CI and code style (prettier, eslint, unit tests and github actions) IHMO

Yes, it would be nice to have prettier, esline, unit tests, and GitHub Actions set up.

> Are you working on switching to axios or I can propose a first small PR?

I'm not working on anything at the moment. Switching to [axios](https://www.npmjs.com/package/axios) would be great. Put it on the roadmap and submit a small PR.

**Comment by friedrith** [comment]
Ok I saw a project 3.0.0 on your repository with a ticket with about replacing request by axios opened on May 7, 2018 so it was in progress.
![image](https://user-images.githubusercontent.com/4005226/89823208-17eb7f00-db1f-11ea-9bf8-ca0a90ce4b36.png)

But ok I will start a PR.

**Comment by karbassi** [comment]
Ah, yes. That's old. I just forgot about it.

**Comment by solick** [comment]
@karbassi just wanted to ask if the project is still maintained. If not it might be better to set it to unmaintained so that everyone knows.

**Comment by floetenbaer** [comment]
> @floetenbaer I look forward to your PR and hopefully maintaining the project.

@karbassi If you like, I can assist you by maintaining this repro to keep it up-to-date.

**Comment by karbassi** [comment]
> @karbassi just wanted to ask if the project is still maintained. If not it might be better to set it to unmaintained so that everyone knows.

As of right now, it is semi-maintained. I'm more than willing to maintain the codebase, I just don't have the devices anymore. If anyone is willing to test my PRs and/or willing to share their keys (privately) with me to test with, that would be great. 

I reached out to [Netatmo](https://github.com/Netatmo) a while ago via email and twitter, but no one responded. 

cc: @ArtemisMucaj @fabienduval

**Comment by solick** [comment]
> > @karbassi just wanted to ask if the project is still maintained. If not it might be better to set it to unmaintained so that everyone knows.
> 
> As of right now, it is semi-maintained. I'm more than willing to maintain the codebase, I just don't have the devices anymore. If anyone is willing to test my PRs and/or willing to share their keys (privately) with me to test with, that would be great.
> 
> I reached out to [Netatmo](https://github.com/Netatmo) a while ago via email and twitter, but no one responded.
> 
> cc: @ArtemisMucaj @fabienduval

@karbassi why don't add @floetenbaer as maintainer so that he can take care of tests and PR?

**Comment by karbassi** [comment]
> @karbassi why don't add @floetenbaer as maintainer so that he can take care of tests and PR?

I've already sent an invitation out to them.

**Comment by floetenbaer** [comment]
> > @floetenbaer I look forward to your PR and hopefully maintaining the project.
> 
> @karbassi If you like, I can assist you by maintaining this repro to keep it up-to-date.

@karbassi : Invitation is revoked: "Sorry, we couldn't find that repository invitation. It is possible that the invitation was revoked or that you are not logged into the invited account."

**Comment by karbassi** [comment]
> > > @floetenbaer I look forward to your PR and hopefully maintaining the project.
> >  
> > @karbassi If you like, I can assist you by maintaining this repro to keep it up-to-date.
> 
> @karbassi : Invitation is revoked: "Sorry, we couldn't find that repository invitation. It is possible that the invitation was revoked or that you are not logged into the invited account."

@floetenbaer weird. Let me try again.
